NitroxModel/Helper/SteamFinder.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.Win32;
+using NitroxModel.Logger;
+using System;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace NitroxModel.Helper
+{
+    public class SteamFinder
+    {
+        public static string FindSteamGamePath(int appid, string gameName)
```

Comment:
Can you return an `Optional<string>` or throw an exception when the path cannot be resolved from the registry? This to properly indicate that it isn't available.